mkdocs.yml

```diff
@@ -58,7 +58,7 @@ plugins:
   
 # Page tree
 nav:
-    - About Orchard Core: README.md
+    - About Orchard Core: docs/README.md
```

Comment:
The ./docs/getting-started/templates/ link in it should now rather be getting-started/templates/README.md

Reply:
No, the link to templates is correct starting by docs/ as the other ones.

Reply:
The readme is under src\docs\README.md. The URL "./docs/getting-started/templates/" (note a single leading dot) refers to src\docs\docs\getting-started\templates which doesn't exist. If you check the link rendered on https://github.com/OrchardCMS/OrchardCore/tree/ag/docsProject/src/docs this is also what happens and the link is broken. That's why I'm saying that, including adding the README.md reference as I've done under https://github.com/OrchardCMS/OrchardCore/pull/6146, the URL of that link should be "getting-started/templates/README.md".

I'm just commenting on it here because I can't do it on the docs/README.md file.

Reply:
ok, fixed.